.changeset/crueler-beers-also-retire.md

```diff
@@ -0,0 +1,8 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+
+tree: 'nodeChanged' and 'treeChanged' events on nodes
```

Comment:
@alexvy86 Since this is a new changeset in this PR it won't automatically link back to the change - can you add a link to the PR or the commit that contained the original change?

Reply:
Sure, where exactly do we need it? I don't see links in the md file, but if something during CI will add them, where should I put it? Or you mean just in the PR for reference?

Reply:
Add it the the changeset body. Something like, "This change was originally made in #1234."

Reply:
https://github.com/microsoft/FluidFramework/blob/main/packages/dds/tree/CHANGELOG.md is an example of how the auto-added PR/commit links look. In this case those will point to this PR/commit. Best we can do is link to the "real" one in the body.

Reply:
Added